barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/databus.hpp

```diff
@@ -80,6 +80,16 @@ struct DatabusPropagationData {
 
     // Is this a kernel circuit (used to determine when databus consistency checks can be appended to a circuit in IVC)
     bool is_kernel = false;
+
```

Comment:
I wanted to print these out, and I thought why not leave this function in place (it lets you write `info` on a `DatabusPropagationData`).